movement/movement_config.h

```diff
@@ -39,4 +39,11 @@ const watch_face_t watch_faces[] = {
 
 #define MOVEMENT_NUM_FACES (sizeof(watch_faces) / sizeof(watch_face_t))
 
+/* Determines what face to go to from the first face if you've already set 
+ * a mode long press to go to the first face in preferences, and
+ * excludes these faces from the normal rotation.
+ * Usually it makes sense to set this to the preferences face.
+ */
+#define MOVEMENT_SECONDARY_FACE_INDEX (MOVEMENT_NUM_FACES - 2)
```

Comment:
If we merge this, I would like to have this #defined to 0 by default. The standard F-91W has its time settings screen in rotation, and all the documentation describes settings and time set as always being available as the last two watch faces. Changing the default behavior to hide the settings screens represents a huge change to the UX.